aries-site/src/pages/components/button.mdx

```diff
@@ -49,6 +49,26 @@ The HTML elements for buttons and links (a.k.a. anchors) describe a very specifi
 - Use buttons when you are performing an action, such as: "submit," "merge," "create new," "upload," etc.
 - An action almost always occurs on the same page
 
+### Toggle Buttons
+
+Toggle buttons are used either to switch between two states, such as check and unchecked, or to trigger two actions like on or off.
+
+#### Toggle Buttons with Icons
+
+Icon buttons have an aria-label that will describe the action that the user will be performing when using screen readers.
+
+**Best Practices:**
+- Limit to 2-3 descriptive words.
+- Use verbs first to describe the action such as “Toggle” , “Favorite” or “Save”.
```

Comment:
The code in ThemeModeToggle uses the verb "Switch".  Maybe that should be mentioned in this list.

In the context of toggle buttons with icons, "Favorite" and "Save" as verbs don't fit here.  It needs the two toggle states with associated icons. E.g. "Like/Dislike", "Light/Dark", "Hide/View".